### DISCUSSION TITLE
Navigation via URL button (or other control) available from MASS screens

### DISCUSSION THREAD

**Discussion by FrankJaco** [music-assistant/support#885] [Feature requests and ideas]
This feature request is especially important when using the HA companion app on your mobile phone. I am looking for a graceful programmable way to exit from a MASS screen to the URL of my choice.  If not every MASS screen, at least have one at the main screen. It could look like all your other buttons on the main screen (Artists. Albums, Tracks, Radio, Playlists, Browse).  Add one is at first labeled "Jump to'.  Make the icon and the label choice programmable and give it the ability to navigate an URL.
![image](https://user-images.githubusercontent.com/79318542/187253054-bfbdd8d8-156c-455a-a7f6-f975deabbff5.png)

I would use it to immediate jump to my custom Mediaplayer view of my active speakers. Others may simply want to directly go to the their HA home page.
![image](https://user-images.githubusercontent.com/79318542/187254637-5e45cabb-9174-46ad-babc-d3b0b5040567.png)

fj

**Comment by OzGav** [comment]
Relates to v1
